pos_transfer_account/readme/DESCRIPTION.md

```diff
@@ -0,0 +1,12 @@
+This module was written to extend the functionality of odoo Point Of Sale about
+transfer account.
+
+This module is usefull only in a multi Point of Sale context of multi company
+context.
+
+With this module, it is now allowed to define for each PoS config a transfer
+account.
+
+If this account is set, it will be used when user realizes "Put Money In" or
+"Take Money Out" operation, instead of the default one set in 'Invoicing' /
+'Configuration' / 'Setting' Section. ("Inter-Banks Transfer Account" field)
```

Comment:
It's `Internal Transfer` in 18.0
<img width="1149" height="686" alt="image" src="https://github.com/user-attachments/assets/5884a0c4-4951-4595-ae12-efcdbc166344" />